packages/js/client/src/__tests__/core/uri-resolution/uri-resolution.spec.ts

```diff
@@ -360,23 +360,22 @@ describe("URI resolution", () => {
     const fromUri = new Uri(`ens/from.eth`);
     const redirectUri = new Uri(`ens/to.eth`);
 
-    const client = await getClient();
+    const client = await getClient({
```

Comment:
This is an invoke-time reconfiguring test, so we should rename the test and test reconfigure here

Reply:
I'm not too keen on the whole `getClient(Partial<ClientConfig>)` thing either... Probably something we should work to remove now that we have the ClientConfigBuilder + reconfigure.